src/network/networkd-network.c

```diff
@@ -1252,3 +1274,15 @@ static const char* const ipv6_link_local_address_gen_mode_table[_IPV6_LINK_LOCAL
 
 DEFINE_STRING_TABLE_LOOKUP(ipv6_link_local_address_gen_mode, IPv6LinkLocalAddressGenMode);
 DEFINE_CONFIG_PARSE_ENUM(config_parse_ipv6_link_local_address_gen_mode, ipv6_link_local_address_gen_mode, IPv6LinkLocalAddressGenMode, "Failed to parse IPv6 link local address generation mode");
+
+static const char* const activation_policy_table[_ACTIVATION_POLICY_MAX] = {
+        [ACTIVATION_POLICY_UP] = "up",
+        [ACTIVATION_POLICY_ALWAYS_UP] = "always-up",
+        [ACTIVATION_POLICY_MANUAL] = "manual",
+        [ACTIVATION_POLICY_ALWAYS_DOWN] = "always-down",
+        [ACTIVATION_POLICY_DOWN] = "down",
+        [ACTIVATION_POLICY_BOUND] = "bound",
```

Comment:
Not a matter, but I'd like to align these entries.

Reply:
ack